Server/Utilities/CBXDevice.m

```diff
@@ -286,7 +286,13 @@ - (NSDictionary *)formFactorMap {
       @"iPhone14,8" : @"iPhone 14 Plus",
       @"iPhone15,2" : @"iPhone 14 Pro",
       @"iPhone15,3" : @"iPhone 14 Pro Max",
-      
+
+      // iPhone 15 Series.
```

Comment:
two lines above this comment seem to be 15 series too.

Reply:
No, that's correct.